### ISSUE TITLE
Publish to PyPI

### ISSUE THREAD

**Issue by luizirber** [Lab41/hypothesis-bio#90]
Who should I add as maintainer to https://pypi.org/project/hypothesis-bio/?

I archived https://github.com/luizirber/hypothesis-bio, so everything is ready for this repo to take it over on PyPI too.

**Comment by vaastav** [comment]
Probably @Benjamin-Lee

**Comment by Benjamin-Lee** [comment]
Yes me and @mbhall or @vaastav if they want. I’m going to work on this more when I’m no longer dying with PhD apps 😰

**Comment by konstin** [comment]
Any news on this? It would be great to have an up-to-date version on pypi
